avocado/core/job.py

```diff
@@ -291,7 +291,8 @@ def _make_test_suite(self, references=None):
         """
         loader.loader.load_plugins(self.args)
         try:
-            suite = loader.loader.discover(references)
+            force = getattr(self.args, 'ignore_missing_references', 'off')
```

Comment:
I was about to be pedantic about the variable (and parameter) name, but this is pretty much internal, so let's keep it.